testsuites/CBLTester/upgrade_tests/test_cbl_upgrade.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+import os
+
+from CBLClient.Database import Database
+from keywords.utils import log_info
+from CBLClient.Replication import Replication
+from keywords.MobileRestClient import MobileRestClient
+from CBLClient.Authenticator import Authenticator
+from libraries.testkit.cluster import Cluster
+from couchbase.n1ql import N1QLQuery
+from couchbase.bucket import Bucket
+from keywords.constants import SDK_TIMEOUT
+import time
+from CBLClient.MemoryPointer import MemoryPointer
+from keywords.couchbaseserver import CouchbaseServer
+
+
+@pytest.mark.listener
+@pytest.mark.upgrade_test
+def test_upgrade_cbl(params_from_base_suite_setup):
+    base_liteserv_version = params_from_base_suite_setup["base_liteserv_version"]
+    upgraded_liteserv_version = params_from_base_suite_setup["upgraded_liteserv_version"]
+    liteserv_platform = params_from_base_suite_setup["liteserv_platform"]
+    liteserv_host = params_from_base_suite_setup["liteserv_host"]
+    upgrade_cbl_db_name = "upgarded_db"
+    base_url = params_from_base_suite_setup["base_url"]
+    encrypted_db = params_from_base_suite_setup["encrypted_db"]
+    db_password = params_from_base_suite_setup["db_password"]
+    sg_db = "db"
+    sg_admin_url = params_from_base_suite_setup["sg_admin_url"]
+    sg_blip_url = params_from_base_suite_setup["target_url"]
+    sg_version = params_from_base_suite_setup["sync_gateway_version"]
+    server_version = params_from_base_suite_setup["server_version"]
+    cluster_config = params_from_base_suite_setup["cluster_config"]
+    sg_config = params_from_base_suite_setup["sg_config"]
+    cbs_ip = params_from_base_suite_setup["cbs_ip"]
+    utils_obj = params_from_base_suite_setup["utils_obj"]
+    server_url = params_from_base_suite_setup["server_url"]
+
+    supported_base_liteserv = ["1.4", "2.0.0", "2.1.5", "2.5.0"]
+    db = Database(base_url)
+    if encrypted_db:
+        if base_liteserv_version < "2.1.5":
```

Comment:
2.1 .0 should also have it

Reply:
don't have app for it, also doesn't matter as 2.1.5 is minor releases. I checked with Jim and he told 2.1.5, will cover 2.1.0